port/unix/protect_helpers.c

```diff
@@ -45,6 +46,12 @@ struct {
 intptr_t
 protect_memory(struct OMRPortLibrary *portLibrary, void *address, uintptr_t length, uintptr_t flags)
 {
+#if defined(OMRZTPF)
+	/* Return success, zTPF does not support mprotect. */
+	IDATA rc = 0;
+
+	return rc;
```

Comment:
Our intention was to no-op the operation, we didn't want to fail JVM operations as I believe a while ago architects deemed it was okay for now if we didn't support it.